google-cloud-storage/src/main/java/com/google/cloud/storage/ParallelCompositeUploadBlobWriteSessionConfig.java

```diff
@@ -534,8 +566,40 @@ private WithPrefix(SecureRandom rand, String prefix) {
       }
 
       @Override
-      protected String fmtFields(String randomKey, String nameDigest, String partRange) {
+      protected String fmtFields(String randomKey, String ultimateObjectName, String partRange) {
```

Comment:
~Shouldn't this implementation of `fmtFields()` be left alone?~

ah before fmtFields() would receive a digest instead of the ultimateObject name.